web/src/info_overlay.js

```diff
@@ -20,105 +20,217 @@ import * as util from "./util";
 // set_up_toggler is called.
 export let toggler;
 
+function format_usage_html(...keys) {
+    const get_formatted_keys = () => keys.map((key) => `<kbd>${key}</kbd>`).join("+");
+    return $t_html(
+        {
+            defaultMessage: "(or <key-html></key-html>)",
+        },
+        {
+            "key-html": get_formatted_keys,
+        },
+    );
+}
+
 const markdown_help_rows = [
     {
-        markdown: "**bold**",
-        usage_html: "(or <kbd>Ctrl</kbd>+<kbd>B</kbd>)",
+        markdown: $t(
+            {
+                defaultMessage: "{markdown_syntax}bold{markdown_syntax}",
+            },
+            {markdown_syntax: "**"},
+        ),
+        usage_html: format_usage_html("Ctrl", "B"),
     },
     {
-        markdown: "*italic*",
-        usage_html: "(or <kbd>Ctrl</kbd>+<kbd>I</kbd>)",
+        markdown: $t(
+            {
+                defaultMessage: "{markdown_syntax}italic{markdown_syntax}",
+            },
+            {markdown_syntax: "*"},
+        ),
+        usage_html: format_usage_html("Ctrl", "I"),
     },
     {
-        markdown: "~~strikethrough~~",
+        markdown: $t(
+            {
+                defaultMessage: "{markdown_syntax}strikethrough{markdown_syntax}",
+            },
+            {markdown_syntax: "~~"},
+        ),
     },
     {
-        markdown: ":heart:",
+        markdown: $t(
+            {
+                defaultMessage: "{markdown_syntax}heart{markdown_syntax}",
```

Comment:
I'm not sure if we need to translate this, because you can't create an emoji with some translated text between ":" syntax.

Reply:
I've decided that it shouldn't be translated; otherwise, the syntax will be broken.